content-src/components/Header/Header.js

```diff
@@ -33,6 +36,13 @@ const Header = React.createClass({
         </ul>
       </section>
       <section className="spacer" />
+      <section className="filter">
+        <input
+          onChange={e => props.dispatch(actions.NotifyFilterQuery(e.target.value))}
+          placeholder="Filter your activity stream"
```

Comment:
Changed to Search and capitalized Activity Stream. Also fixed the sizing/positioning as the longer placeholder text got cut off.